sag_12r.par

```diff
@@ -1,33 +1,33 @@
-! Sagittal 12R notation. Dave Keenan & George Secor. 2006-Mar-22, updated 2012-Sep-05
-! 
+! Sagittal 12R notation. Dave Keenan & George Secor. 2006-Mar-22, updated 2020-Apr-08
+!
 max_abs_slope	6.0
 c_exp_range	-6	10
 nc_exp_range	-1	5
 fifth_size	700.0
 oct_size	2/1
 !
 !Lower bound	Symbol	Value		Short	Most common semitone fraction value
--57.14285715	(!/	8192/8505	w	-5/8
+-57.14285715	(!/	8192/8505	d	-5/8
 -51.99999999	\!/	32/33		v	-1/2
--48.00000001	\!)	35/36		u	-4/9
--42.85714285	\!~	45/46		�	-2/5
+-48.00000001	\!)	35/36		&	-4/9
+-42.85714285	\!~	45/46			-2/5
 -38.88888888	!/	54/55		k	-3/8
 -33.33333334	!)	63/64		t	-1/3
--28.57142856	!~	2025/2048	z	-1/4
--22.22222223	(!	3584/3645	j	-1/5
+-28.57142856	!~	729/736	z	-1/4
```

Comment:
I updated these to match the JI, unless there's something I don't know and these four entries are supposed to have different primary values in Trojan.

Reply:
They were not supposed to have different primary values (that's almost a contradiction in terms). They were supposed to have secondary values. But I've just realised they are not valid secondaries. See http://forum.sagittal.org/viewtopic.php?p=1726#p1726

Reply:
Per the forum topic linked above, we've decided the values to which I changed these two symbols are correct. Resolving.